sdk/src/test/java/io/opentelemetry/sdk/resource/ResourceTest.java

```diff
@@ -148,4 +150,15 @@ public void testMergeResources_Resource2_Null() {
     Resource resource = DEFAULT_RESOURCE.merge(resource1).merge(null);
     assertThat(resource.getAttributes()).isEqualTo(expectedAttributes);
   }
+
+  @Test
+  public void testSdkTelemetryResources() {
+    Resource resource = Resource.getTelemetrySdk();
+    ReadableAttributes attributes = resource.getAttributes();
+    String[] keys = {"telemetry.sdk.name", "telemetry.sdk.language", "telemetry.sdk.version"};
```

Comment:
Think it's better to avoid the forEach

```java
assertThat(attributes.get("telemetry.sdk.name")).isEqualTo(stringAttributeValue("opentelemetry"));
assertThat(attributes.get("telemetry.sdk.language")).isEqualTo(stringAttributeValue("java"));
assertThat(attributes.get("telemetry.sdk.version")).isNotNull();
```

/cc PS @jkwatson Would love to have `getString` :D

Reply:
I'be initially against adding too many public methods to `Attributes`, as this is in general a class expected to be used by the SDK processing, not by end users. `get()` IMHO is enough for this purpose (and having to write more code in *our* side won't kill us ;) )

Reply:
This test could be made simpler by just building the Attributes object you expect and asserting that what you get back is equal to it.